assets/src/edit-story/components/popup/index.js

```diff
@@ -65,29 +92,166 @@ const Container = styled.div.attrs(({ x, y }) => ({
   }
 `;
 
-function Popup({ anchor, children, width = DEFAULT_WIDTH, isOpen }) {
+function getXTransforms(placement) {
```

Comment:
Maybe create an `util.js` file for popup and move all the helpers there? Would make it possible to unit test better as well. Thoughts?